kustomize/overlays/sandbox/kustomization.yaml

```diff
@@ -17,13 +17,11 @@ bases:
 # Contour is used to create ingress. On cloud service use the default provided ingress controllers or cloud LB's
 # Add node ports for ease of use locally
 resources:
-  - ../../base/operators/spark
-  - ../../base/operators/kfoperators/pytorch
   - ./dependencies/database
   - ./dependencies/storage
-  - ./dependencies/redis
   - ./dependencies/contour_ingress_controller
   - ./dependencies/nodeport-services.yaml
+  - ../../base/namespace
```

Comment:
This is ok for getting this to work, but should really be in `single_cluster/headless`, right?

Reply:
Ideally yes, if it were not for pytorch deploying their own namespace and the kustomize namespace override thing not playing nice with it... Somebody [wrote a transformation for it](https://github.com/kubernetes-sigs/kustomize/issues/880) that we might want to experiment with at some point...